flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  description = "go lang, SQLite, ollama, deepseek-r1 model";
```

Comment:
I'll remove the deepseek-r1 model, since that looks like it was left as a mistake

Reply:
https://github.com/tlehman/git-llama/commit/dc3b2ef0641539a3737a4812078153f6cf6e3b7d